langchain/src/agents/initialize.ts

```diff
@@ -1,17 +1,27 @@
-import { Tool } from "./tools/index.js";
 import { BaseLLM } from "../llms/index.js";
 import { AgentExecutor } from "./executor.js";
-import { ZeroShotAgent } from "./mrkl/index.js";
+import { CreatePromptArgs, ZeroShotAgent } from "./mrkl/index.js";
+import { Tool } from "./tools/index.js";
+
+export interface InitializeAgentExecutorOptions {
+  tools: Tool[];
+  llm: BaseLLM;
+  agentType?: "zero-shot-react-description";
+  promptArgs?: CreatePromptArgs;
```

Comment:
will probably have to be made a bit more general to account for different types of agents